doc/ref/cli/salt-key.rst

```diff
@@ -104,10 +104,20 @@ Actions
 
     Reject the specified public key (use --include-all to match accepted keys
     in addition to pending keys). Globs are supported.
+    .. warning::
+        Using globs to reject the specified public key may cause performance issues
+        because of a Salt master degredation dramatic performance salt-master degradation. 
+        See [`performance guide`](https://docs.saltproject.io/en/latest/topics/tutorials/intro_scale.html#too-many-minions-re-authing) 
+        for more information about performance impacts and workarounds.
```

Comment:
This needs to be reworded:
   Using globs to reject public keys may degrade Salt master performance.  
   See the [`performance guide`](https://docs.saltproject.io/en/latest/topics/tutorials/intro_scale.html#too-many-minions-re-authing)  
   for details on performance impacts and possible workarounds.